### PR TITLE
Introducing haml

### PR DESCRIPTION
**Pull request by sllewelyn-mdsol**
# Feature

haml is a gem that replaces .erb syntax with syntax way more comfortable for rubyists.  See the tutorial here: http://haml.info/development.html

In this PR, I have haml'd the sign_in page.  Todo: haml the rest of the views

# Screenshots

No visual change to the view page (success)
image

# Tests

rspec spec/
```
Finished in 2.97 seconds (files took 1.97 seconds to load)
19 examples, 0 failures, 1 pending
```

### CODE REVIEW

File: app/views/home/sign_in.html.haml [hunk 1]
```diff
@@ -0,0 +1,23 @@
+.sign-in
```
Comment (sllewelyn-mdsol) [inline app/views/home/sign_in.html.haml]: Equivalent to <div class='sign-in'> ...

File: app/views/home/sign_in.html.haml [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  %header
```
Comment (sllewelyn-mdsol) [inline app/views/home/sign_in.html.haml]: Equivalent to <header>...

File: app/views/home/sign_in.html.haml [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+        %td
+          %span.big-number 2
+        %td Send your contact info to other hosts and travelers
+      %tr
+        %td
+          %span.big-number 3
+        %td Confirm with text messages / phone calls.
+  %section.sign-up
+    %h1 Sign In Using...
+    = link_to "Facebook", "/auth/facebook"
```
Comment (sllewelyn-mdsol) [inline app/views/home/sign_in.html.haml]: Equivalent to ```<%= blah blah ruby code ...```

**Comment by PepperTeasdale** [comment]
Seems worthwhile. Would you be able to try and integrate some of the bootstrap while doing so?

**Comment by sllewelyn-mdsol** [comment]
Yeah we can do bootstrap a future PR.  They're not mutually exclusive.  & everything you can do in erb can be done in .haml files.  

<3 small PRs

**Pull request merged**